packages/playground/src/app/basic-page/basic-page.component.ts

```diff
@@ -12,7 +13,7 @@ import { TodosService } from '../todos.service';
   template: `
     <h2 class="mb-3">Todos</h2>
 
-    <ng-container *subscribe="todos$ as todos">
+    <ng-container *ngIf="todos$ | async as todos">
```

Comment:
Why did you change it?

Reply:
For some reason I had errors in IDE that properties weren't recognized. Will revert.